torch/_tensor_docs.py

```diff
@@ -3202,10 +3238,39 @@ def callable(a, b) -> number
                r"""
 sparse_dim() -> int
 
-If :attr:`self` is a sparse COO tensor (i.e., with ``torch.sparse_coo`` layout),
+If :attr:`self` is a :ref:`sparse tensor <sparse-docs>`,
 this returns the number of sparse dimensions. Otherwise, this throws an error.
 
-See also :meth:`Tensor.dense_dim`.
+See also :meth:`Tensor.dense_dim` and :ref:`hybrid tensors <sparse-hybrid-docs>`.
+""")
+
+add_docstr_all('sparse_resize_',
+               r"""
+sparse_resize_(size, sparse_dim, dense_dim) -> Tensor
+
+Resizes :attr:`self` :ref:`sparse tensor <sparse-docs>` to the
```

Comment:
"Resizes a sparse tensor self ... "

The one sentence should incorporate references to the required arguments size, sparse_dim, and dense_dim. What does non-shrinking mean?

Reply:
non-shrinking means that one can increase the values of the dimensions but not decrease.

Now using:
```
Resizes :attr:`self` :ref:`sparse tensor <sparse-docs>` to the desired
size and the number of sparse and dense dimensions if :attr:`self` is
empty tensor.

Resizes :attr:`self` :ref:`sparse tensor <sparse-docs>` to the larger
or equal size if :attr:`self` is a non-empty tensor. The specified
sparse and dense dimensions must be equal to the sparse and dense
dimensions of :attr:`self`.

Throws an error if :attr:`self` is not a sparse tensor.
```